staking/network/values.go

```diff
@@ -0,0 +1,15 @@
+package network
+
+import (
+	"math/big"
+
+	"github.com/harmony-one/harmony/numeric"
+)
+
+// UtilityMetric ..
+type UtilityMetric struct {
```

Comment:
Why its not even used in staking/network/reward.go? And its not easy to understand how results returned by methods in reward.go correspond to UtilityMetric.